ZUGFeRD/InvoiceDescriptor22UBLWriter.cs

```diff
@@ -64,12 +64,13 @@ public override void Save(InvoiceDescriptor descriptor, Stream stream, ZUGFeRDFo
 
             Writer.WriteStartDocument();
 
-            if (this.Descriptor.Type != InvoiceType.Invoice && this.Descriptor.Type != InvoiceType.CreditNote)
+
+            if (this.Descriptor.Type != InvoiceType.Invoice && this.Descriptor.Type != InvoiceType.CreditNote && this.Descriptor.Type != InvoiceType.Correction)
```

Comment:
can you please add brackets around the parts of the if clause (consistent code style)?